check.go

```diff
@@ -113,6 +114,13 @@ func FilterIgnorePath(files []string, pattern string) ([]string, error) {
 func FilterPath(files []string, pattern string) ([]string, error) {
 	var out []string
 	for _, file := range files {
+		// Check for a prefix match
+		if strings.HasPrefix(file, pattern) {
```

Comment:
This could possibly be on line [128](https://github.com/telia-oss/github-pr-resource/blob/2d8646faa4b1f36ed545e6c8b8fb233f2ec0af94/check.go#L128) as:
```go
if match || strings.HasPrefix(file, pattern) {
  // append...
}
```